config/src/main/java/com/typesafe/config/impl/Tokenizer.java

```diff
@@ -52,6 +52,14 @@ else if (Character.isISOControl(codepoint))
         return new TokenIterator(origin, input, flavor != ConfigSyntax.JSON);
     }
 
+    static String render(Iterator<Token> tokens) {
+        String renderedText = "";
```

Comment:
hmm, no that's probably not worth mixing in to this PR. can leave it as is for now.